src/resources/audio.js

```diff
@@ -34,15 +34,16 @@ const ie = (function () {
     return false;
 })();
 
-const toMIME = {
-    '.ogg': 'audio/ogg',
-    '.mp3': 'audio/mpeg',
-    '.wav': 'audio/x-wav',
-    '.mp4a': 'audio/mp4',
-    '.m4a': 'audio/mp4',
-    '.mp4': 'audio/mp4',
-    '.aac': 'audio/aac'
-};
```

Comment:
What's the reason for changing to use extensions instead of MIMEs?

Reply:
The mime wasn't used at all. That's why I also renamed it to `supportedExtensions`, because it was only used to do a lookup if the extension was supported.